enrolments/serializers.py

```diff
@@ -56,12 +56,16 @@ class ClassCreateSerializer(serializers.HyperlinkedModelSerializer):
     facilitator = serializers.PrimaryKeyRelatedField(
         queryset=User.objects.all(), allow_null=True
     )
+    session = serializers.PrimaryKeyRelatedField(
+        queryset=Session.objects.all(), allow_null=True, required=False
```

Comment:
I noticed we have `required=False`, is there a reason why a class wouldn't have a session?

Reply:
By making required=True, it'll create a circular dependency when we create a session with classes since the classes session isn't yet created